Tribler/community/chant/chant.py

```diff
@@ -0,0 +1,129 @@
+import os
+from datetime import datetime
+
+from libtorrent import add_files, bdecode, bencode, create_torrent, file_storage, set_piece_hashes
+from Tribler.community.chant.MDPackXDR import serialize_metadata_gossip, \
+    deserialize_metadata_gossip, CHANNEL_TORRENT, MD_DELETE
+
+from Tribler.community.chant.orm import MetadataGossip, Peer, known_sig, known_pk, trusted_pk
+
+channels_dir = "/tmp"
+
+
+def create_torrent_from_dir(directory, torrent_filename):
+    fs = file_storage()
+    add_files(fs, directory)
+    t = create_torrent(fs, flags=19)  # ??? 19 ???
```

Comment:
This is the only way to backwards compatible set all the required flags for a merkle torrent which takes care of efficient updating of files. The required flags were renamed between libtorrent versions.

Reply:
I'll put that into the comments.

Reply:
https://www.libtorrent.org/reference-Create_Torrents.html#enum-flags-t (optimize_alignment, merkle, mutable_torrent_support)

Reply:
@qstokkink , thanks.

Reply:
Also, you need to set the piece size here to be able to align to a piece size and make everything efficient.

Reply:
@qstokkink, I removed that line because for testing purposes I write each entry into its own file now. When the code will become more mature I will start squashing entries together into chunks, and then the piece size setting will be needed.